app/models/foreman_tasks/recurring_logic.rb

```diff
@@ -0,0 +1,53 @@
+module ForemanTasks
+
+  require 'parse-cron'
+
+  class RecurringLogic < ActiveRecord::Base
+    include Authorizable
+
+    has_one :task_group,
+            :class_name => TaskGroups::RecurringLogicTaskGroup.name
+
+    delegate :tasks, :to => :task_group
```

Comment:
something line `has_many :tasks, :through => :task_group` would be better, as it leverages the activerecord AREL